languages/go/INSTRUCTIONS.md

```diff
@@ -0,0 +1,46 @@
+# Instructions
+
+This guide is for developers who want to use the Bitwarden Go SDK module in their own Go projects.
+
+The Bitwarden SDK is written in Rust, and we utilize FFI via [cgo](https://pkg.go.dev/cmd/cgo). We currently support static linking the Bitwarden Go module on Linux (x86-64 & arm64), Mac (x86-64 & arm64), and Windows (x86-64).
+
+Please see the main [README](./README.md) and [example](./example/example.go) for examples.
+
+## Linux
+
+1. Make sure you have the following on your system:
+    - [Go](https://go.dev/dl)
+    - A C/C++ toolchain. We recommend the [MUSL toolchain](https://musl.libc.org). You can install this on most debian based systems with: `sudo apt install musl-tools`
+2. Verify cgo is enabled and the `CC` and `CXX` flags are set with: `go env`.
+    - You can enable cgo by setting the flag: `go env -w CGO_ENABLED=1`
+    - If `CC` and `CXX` are not set, you can set them with: `go env -w CC=clang CXX=clang++`
```

Comment:
```suggestion
    - If `CC` is not set, you can set it with: `go env -w CC=musl-gcc`
```

Most Linux distros don't use Clang by default. We need to use `musl-gcc` for `CC`. I don't see an equivalent `musl-g++`, but leaving `CXX` unset or the default is fine for our usage.